tests/test_pytorch_dataloader_api.py

```diff
@@ -187,6 +187,7 @@ def test_dataset_generator_batch_output(
                 tiledb_dataset, batch_size=None, num_workers=workers
             )
 
+            next(iter(train_loader))
```

Comment:
Is this leftover? If not it merits a comment

Reply:
It is you are right I will fix. It is a leftover when I was checking the why the `mixin` was not working properly when I had the executor as an attribute.